docs/using-cinder-csi-plugin.md

```diff
@@ -388,6 +388,10 @@ the cluster in which the plugin is running.
 $ sudo cinder-csi-plugin --endpoint tcp://127.0.0.1:10000 --cloud-config /etc/cloud.conf --nodeid CSINodeID --cluster ClusterID
 ```
 
+#### Rescan block device geometry on in-use volume resize
+
+Some hypervizors (like VMware) don't automatically send a new volume size to a Linux kernel, when a volume is in-use. Sending a "1" to `/sys/class/block/XXX/device/rescan` is telling the SCSI block device to refresh it's information about where it's ending boundary is (among other things) to give the kernel information about it's updated size. When a `rescan-on-resize` flag is set in a CSI node driver cloud-config `[BlockStorage]` section, a CSI node driver will rescan block device and verify its size before expanding the filesystem. CSI driver will raise an error, when expected volume size cannot be detected.
```

Comment:
looks like this path /sys/class/block/XXX/device/rescan is not supported by all hypervisors, I got warning message when tried to access on libvirt. So could we include in doc what all hypervisors could support this rescan

Reply:
Added a note to doc.

Reply:
Thanks

Reply:
@kayrus , quick update , sorry, I haven't noticed before , looks like this paragraph is under Using CSC tool section which is incorrect. Please move this under the section , https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-cinder-csi-plugin.md#volume-expansion